### ISSUE TITLE
Keyboard shortcuts don't behave as expected

### ISSUE THREAD

**Issue by axelsimon** [kewisch/quickmove-extension#184]
Hi and thanks for your work on this super helpful extension!

I have a couple issues with Quickmove's keyboard shortcuts.
- The "open menu" shortcut, `ctrl+shift+N` does nothing. Before I disabled it, CardBook would catch it and open its "new contact" view.
- The "move" shortcut, `ctrl+shift+M` does not open Quickmove menu but moves the currently focused email to the last selected Quickmove target directory (with no feedback or warning, might I add).

This is on Thunderbird 128.5.0esr (Flatpak).

Sorry if these have been reported, I checked and they seem different to me from what is already in the issue tracker :)

**Comment by kewisch** [comment]
The only remaining shortcut should be Ctrl+Shift+N. If this one is taken by CardBook, you have to change it in your Add-ons Shortcut Manager. Go to your add-ons list, click on the cog icon, manage extension shortcuts, and pick a new one that works for you and isn't taken by Thunderbird or another add-on. You can define a shortcut for each action there.

Where did you get Ctrl+Shift+M, did I forget to update documentation somewhere, or muscle memory from before?

**Comment by axelsimon** [comment]
Can't remember where the Ctrl+Shift+M shortcut came from, i think it was suggested at some point?
I also could have come up with it.

In any case, the weirdest part is that the menu doesn't come up at all. Clicking it works, so I know it does work, but the shortcut does nothing.

Right now, CardBook is uninstalled, so it shouldn't create any issues with QuickMove.

**Comment by kewisch** [comment]
Yes Ctrl+Shift+M was an intermediate suggestion, I unfortunately had to change it due to Windows/Linux differences.

What menu do you mean? Here is what it looks like for me:

Hamburger/thee bars menu -> Addons brings you here:

![image](https://github.com/user-attachments/assets/8a2e2e47-8dc0-41ef-8a6a-44e273509b25)

Cog Icon > Manage Extension Shortcut brings you here:

![image](https://github.com/user-attachments/assets/2b07e3b8-49f0-4ff3-871f-90507318ef8a)


And then you say the shortcut you define there does not actually open the menu? Can you double check that it does not conflict with anything else from Thunderbird or an add-on? Sometimes it doesn't say.

**Comment by CharlesDelorme** [comment]
Whoa ! This last message would benefit to be more explicit in the documentation. I was looking for the "manage extension shortcuts" panel within QFM configuration (and obviously never found it) and I really never took time to read the addon menu up to the last item, my bad…

**Comment by CharlesDelorme** [comment]
BTW in 3.0.0 you still have a screen cap with a mac ctrl-shift-M. Since I am a little bit confused as such key combination is only for mac and ctrl-shif-N for win/linux this screen cap may confused others :-)

![image](https://github.com/user-attachments/assets/2753d442-e3ba-4419-900d-330bd6ff3564)

**Comment by CharlesDelorme** [comment]
Here is the screen cap in french under windows
![image](https://github.com/user-attachments/assets/fa389a1d-762c-40a4-9a43-ca9e68be8682)

**Comment by kewisch** [comment]
Do you have a suggestion how to improve the docs? I've updated the README with a bit more info and updated the screenshots. In the onboarding I otherwise link to https://support.mozilla.org/en-US/kb/manage-extension-shortcuts-firefox which admittedly is a Firefox article, but it works the same way in Thunderbird.

@CharlesDelorme by the way, would you be willing to update the French translation at https://hosted.weblate.org/projects/quick-folder-move/main/fr/ ? A few strings are translated from a prior contributor, but it hasn't been updated in a while. I unfortunately don't speak French.

**Comment by CharlesDelorme** [comment]
Concerning the extension shortcuts I must admit I just didn't read everything, so maybe the doc is ok

And I will have a look a the translation

**Comment by CharlesDelorme** [comment]
I gave a try to translations. I only missed one for which I don't now where it might be used ("Please enable the 'Tags' folder view to go to tag folders")

**Comment by kewisch** [comment]
See https://github.com/kewisch/quickmove-extension/issues/182

**Comment by axelsimon** [comment]
> What menu do you mean?

I mean the Quickmove menu:
![Screenshot From 2024-12-18 14-57-20](https://github.com/user-attachments/assets/334d6108-d1c7-4601-b91c-cbf2bf146e13)

But looking at the add-on shortcuts options, I'm starting to think there is no shortcut to just bring up the Quickmove menu anymore, only to trigger QM actions?

**Comment by kewisch** [comment]
No, the shortcuts remain for opening the popup, they simply pre-select the action. Otherwise there would be no way to know which folder to move or go to.

@axelsimon Do you have a shortcut assigned to Move, and if so which is it?
